src/lib/tzcode/localtime.c

```diff
@@ -66,10 +67,10 @@
 #define WILDABBR "   "
 #endif /* !defined WILDABBR */
 
-static const char wildabbr[] = WILDABBR;
-
 static const char gmt[] = "GMT";
 
+static const char wildabbr[] = WILDABBR;
+
 /*
```

Comment:
What for did you change it? Commit "datetime: primary and aliases zones in Olson".

Reply:
It's becoming unused (and compiler complains about it) if there is no `TM_ZONE` activated. And we intentionally disable `TM_ZONE`  as it's incompatible with our own `tnt_tm` where `tm_tzindex` used for referring to the zone name.

Reply:
the same line removed and added back:
![image](https://user-images.githubusercontent.com/1151557/169506592-b0e86f56-7ad1-4705-a1b9-2120ce7bd459.png)

commit "datetime: use Olson for timezone name"